libass/ass_compat.h

```diff
@@ -43,4 +48,199 @@ char *ass_strndup_fallback(const char *s, size_t n); // definition in ass_utils.
 #define strndup ass_strndup_fallback
 #endif
 
+#if defined(_WIN32)
+
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#include <libloaderapi.h>
```

Comment:
Oh, I'd added this at some point while trying to figure out why one of the CI builds wasn't working, but it turned out it was because it's WinRT/UWP. Removed.